package/firmware/omnia-mcu-firmware/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2023 CZ.NIC z.s.p.o. (http://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=omnia-mcu-firmware
+PKG_DISTNAME:=omnia_hw_ctrl
+PKG_VERSION:=3.4
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_SUBDIR:=$(PKG_DISTNAME)-binaries-v$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.bz2
+PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/hw/$(PKG_DISTNAME)/-/releases/v$(PKG_VERSION)/downloads/
+PKG_HASH:=2b0888fb065b279dc5eac90a76fea8e2353bd554b16df07515c4114d6d0a2c8e
+
+PKG_MAINTAINER:=Marek Mojik <marek.mojik@nic.cz>
```

Comment:
Is other Marek aware that he will take care about this package in upstream?

Reply:
Of course :) And if he changes his mind, I will take over.